CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,104 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
-
-	// TODO: Assignment 1
-
+class SpawnedViewSpawnController: UIViewController, UIGestureRecognizerDelegate {
+    
+    
+    private let size: CGFloat = 100
+    
+    //TODO: Move offsets to CircleView
+    var xOffset: CGFloat = 0.0
+    var yOffset: CGFloat = 0.0
```

Comment:
1. Można to trzymać w `CGVector`, albo `CGPoint`.
2. Czemu używacie domyślnie wartości `0.0`? Przecież to w stanie początkowym powinno NIE MIEĆ wartości, czyli powinno być nilem. Od tego są w Swifcie optionale i należy z nich korzystać.
3. Obsługujecie ładne przesuwanie tylko jednego widoku jednocześnie.